src/page/personal-information.page.ts

```diff
@@ -0,0 +1,72 @@
+import { $, by, ElementFinder, ExpectedConditions, browser } from 'protractor';
+import { PersonalInfo } from '../personalInfo';
+
+export class PersonalInformationPage {
+  private firstNameBox: ElementFinder;
+  private lastNameBox: ElementFinder;
+  private sexCheckBox: ElementFinder;
+  private yearsOfExperienceCheckBox: ElementFinder;
+  private professionCheckBox: ElementFinder;
+  private toolSeleniumCheckBox: ElementFinder;
+  private continentSelect: ElementFinder;
+  private commandsSelect: ElementFinder;
+  private submitBtn: ElementFinder;
+  private timeout: number = 8000;
+  private pageTitle: ElementFinder;
+
+  constructor () {
+    this.firstNameBox = $('input[name="firstname"]');
+    this.lastNameBox = $('input[name="lastname"]');
+    this.submitBtn = $('button[name="submit"]');
+    this.pageTitle = $('div.mui-container-fluid.content > div > div.mui-col-md-6.tutorial-content > h1:nth-child(1)');
+  }
+
+  public async finishForm() {
+    await this.submitBtn.click();
+    browser.wait(ExpectedConditions.alertIsPresent(), this.timeout);
+    await browser.switchTo().alert().accept();
+  }
+
+  public async getPageTitle(): Promise<string> {
+    await browser.wait(
+      ExpectedConditions.visibilityOf(this.pageTitle),
+      this.timeout);
+    return this.pageTitle.getText();
+  }
+
+  public async fillForm(info: PersonalInfo): Promise<void> {
+    await browser.wait(
+      ExpectedConditions.visibilityOf(this.firstNameBox),
+      this.timeout);
+
+    await this.firstNameBox.sendKeys(info['firstName']);
```

Comment:
please use dot notation (`info.firstName`). Same applies for the other properties

Reply:
ok! Done